selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/WorkspacesListTest.java

```diff
@@ -215,6 +216,7 @@ private void createWorkspace(String name) {
 
     workspaces.clickOnAddWorkspaceBtn();
     newWorkspace.waitToolbar();
+    newWorkspace.clickOnAllStacksTab();
```

Comment:
please, leave comment about why we can't rely on Quick Start stack list.

Reply:
The **Java** stack, which we use for creating test workspaces, not exist **Quick Start** stack list on **OSIO**.